src/Components/Reports/Common/ExecutiveFirstPage.js

```diff
@@ -75,7 +75,7 @@ export const ExecutiveFirstPage = ({ data }) => {
 
         {/* TODO: update based on api response */}
         {/* eslint-disable-next-line max-len */}
-        <Text>{`6 systems out of a total of ${totalCount} systems have Kernel Pressure Stall Information Enabled. That helps us provide better suggestions either in breakdown of occurance or registered system section`}</Text>
+        <Text>{`6 systems out of a total of ${totalCount} systems have Kernel Pressure Stall Information Enabled. That helps us provide better suggestions either in breakdown of coccurrence or registered system section`}</Text>
```

Comment:
Should this number of systems be hard coded?
Also one typo.

```Suggestion
        <Text>{`6 systems out of a total of ${totalCount} systems have Kernel Pressure Stall Information Enabled. That helps us provide better suggestions either in breakdown of occurrence or registered system section`}</Text>
```

Reply:
@r14chandra I have one question around the same, once thats resolved I will update this section. I have posted about the same in the above similar thread. good catch with the typo I have fixed it. I am resolving this thread now, in case further modifications needed feel free to open this again 💯